src/globalwarmingpotentials/__init__.py

```diff
@@ -9,370 +9,306 @@
-    "C6F14": 7400.0,
-    "C7F16": None,
-    "C8F18": None,
-    "C10F18": None,
-    "SF5CF3": None,
-    "cC3F6": None,
-    "HFE125": None,
-    "HFE134": None,
-    "HFE143a": None,
-    "HCFE235da2": None,
-    "HFE245cb2": None,
-    "HFE245fa2": None,
-    "HFE347mcc3": None,
-    "HFE347pcf2": None,
-    "HFE356pcc3": None,
-    "HFE449sl": None,
-    "HFE569sf2": None,
-    "HFE4310pccc124": None,
-    "HFE236ca12": None,
-    "HFE338pcc13": None,
-    "HFE227ea": None,
-    "HFE236ea2": None,
-    "HFE236fa": None,
-    "HFE245fa1": None,
-    "HFE263fb2": None,
-    "HFE329mcc2": None,
-    "HFE338mcf2": None,
-    "HFE347mcf2": None,
-    "HFE356mec3": None,
-    "HFE356pcf2": None,
-    "HFE356pcf3": None,
-    "HFE365mcf3": None,
-    "HFE374pc2": None,
-    "PFPMIE": None,
-    "CHCl3": 4.0,
-    "CH2Cl2": 9.0,
-    "CH3Cl": None,
-    "Halon1201": None,
-}
 
-AR4GWP100 = {
-    "CH4": 25.0,
-    "N2O": 298.0,
-    "CFC11": 4750.0,
-    "CFC12": 10900.0,
-    "CFC13": 14400.0,
-    "CFC113": 6130.0,
-    "CFC114": 10000.0,
-    "CFC115": 7370.0,
-    "Halon1301": 7140.0,
-    "Halon1211": 1890.0,
-    "Halon2402": 1640.0,
-    "Halon1202": None,
-    "CCl4": 1400.0,
-    "CH3Br": 5.0,
-    "CH3CCl3": 146.0,
-    "HCFC21": None,
-    "HCFC22": 1810.0,
-    "HCFC123": 77.0,
-    "HCFC124": 609.0,
-    "HCFC141b": 725.0,
-    "HCFC142b": 2310.0,
-    "HCFC225ca": 122.0,
-    "HCFC225cb": 595.0,
-    "HFC23": 14800.0,
-    "HFC32": 675.0,
-    "HFC41": None,
-    "HFC125": 3500.0,
-    "HFC134": None,
-    "HFC134a": 1430.0,
-    "HFC143": None,
-    "HFC143a": 4470.0,
-    "HFC152": None,
-    "HFC152a": 124.0,
-    "HFC161": None,
-    "HFC227ea": 3220.0,
-    "HFC236cb": None,
-    "HFC236ea": None,
-    "HFC236fa": 9810.0,
-    "HFC245ca": None,
-    "HFC245fa": 1030.0,
-    "HFC365mfc": 794.0,
-    "HFC4310mee": 1640.0,
-    "SO2F2": None,
-    "SF6": 22800.0,
-    "NF3": 17200.0,
-    "CF4": 7390.0,
-    "C2F6": 12200.0,
-    "C3F8": 8830.0,
-    "cC4F8": 10300.0,
-    "C4F10": 8860.0,
-    "C5F12": 9160.0,
-    "C6F14": 9300.0,
-    "C7F16": None,
-    "C8F18": None,
-    "C10F18": None,
-    "SF5CF3": 17700.0,
-    "cC3F6": None,
-    "HFE125": 14900.0,
-    "HFE134": 6320.0,
-    "HFE143a": 756.0,
-    "HCFE235da2": 350.0,
-    "HFE245cb2": 708.0,
-    "HFE245fa2": 659.0,
-    "HFE347mcc3": 575.0,
-    "HFE347pcf2": 580.0,
-    "HFE356pcc3": 110.0,
-    "HFE449sl": 297.0,
-    "HFE569sf2": 59.0,
-    "HFE4310pccc124": 1870.0,
-    "HFE236ca12": 2800.0,
-    "HFE338pcc13": 1500.0,
-    "HFE227ea": None,
-    "HFE236ea2": None,
-    "HFE236fa": None,
-    "HFE245fa1": None,
-    "HFE263fb2": None,
-    "HFE329mcc2": None,
-    "HFE338mcf2": None,
-    "HFE347mcf2": None,
-    "HFE356mec3": None,
-    "HFE356pcf2": None,
-    "HFE356pcf3": None,
-    "HFE365mcf3": None,
-    "HFE374pc2": None,
-    "PFPMIE": 10300.0,
-    "CHCl3": None,
-    "CH2Cl2": 8.7,
-    "CH3Cl": 13.0,
-    "Halon1201": None,
-}
+def as_data_frame():
```

Comment:
Echoing the equivalent `pandas.Series.to_frame()` method, see [here](https://pandas.pydata.org/docs/reference/api/pandas.Series.to_frame.html)

```suggestion
def as_frame():
```

Reply:
Nice!